lwallet/Pos/Wallet/Light/Mode.hs

```diff
@@ -101,6 +110,20 @@ instance MonadUpdates LightWalletMode where
     waitForUpdate = error "notImplemented"
     applyLastUpdate = pure ()
 
+-- FIXME: Dummy instance for lite-wallet.
```

Comment:
There story is that light wallet requires some constraints which are not used and there many such instances in this file.